src/services/api/Roles/Roles.ts

```diff
@@ -10,7 +10,7 @@ export interface Role extends BaseData {
   startDate: Date;
   startConfidence: number;
   endDate?: Date;
-  endConfidence?: number;
+  endConfidence?: number | null;
```

Comment:
The issue I'm having is if we have a confidence of 0.1 for example and we want to update it to nothing (not 0).

Reply:
ooh. Ok. Null can be "no value" just not a "missing value" (I know, its a minor distinction). As long as this isn't a general pattern of "everywhere we don't have something, we're using `null`" I'm not worried. I've seen projects doing that and it gets very messy very quickly.

Reply:
yeah i spent a few hours trying to solve this. If it was a string we would just send an empty one but to keep the field as a number sending a null is the only thing that worked for me and to avoid some cases where the data comes back with a 0 instead of nothing